packages/grafana-ui/src/components/Table/TableNG/utils.tsx

```diff
@@ -625,3 +627,22 @@ export function withDataLinksActionsTooltip(field: Field, cellType: TableCellDis
     (field.config.links?.length ?? 0) + (field.config.actions?.length ?? 0) > 1
   );
 }
+
+export function maybeWrapWithLink(field: Field, rowIdx: number, children: ReactNode): ReactNode {
```

Comment:
We had discussions behind the scenes that this obviously should just be a Component, but @leeoniya disagreed. I do however want to make sure we basically _never_ follow this pattern of returning JSX from utils anywhere else, so please add a comment explaining why this isn't a component. (That also helps some enterprising engineer understand why we did this weird thing)

Reply:
:+1:  switched to helper component instead